providers/dns/dns_providers.go

```diff
@@ -202,6 +203,8 @@ func NewDNSChallengeProviderByName(name string) (challenge.Provider, error) {
 		return gandi.NewDNSProvider()
 	case "gandiv5":
 		return gandiv5.NewDNSProvider()
+	case "googledomains":
+		return googledomains.NewDNSProvider()
```

Comment:
the providers are sorted alphabetically. Can you move this part to the right position?

Reply:
Yep! Renamed the provider as I was working on it so I forgot :/